server/core/store_option.go

```diff
@@ -65,6 +65,20 @@ func SetStoreUnBlock() StoreCreateOption {
 	}
 }
 
+// SetStoreOverload stops balancer from selecting the store.
+func SetStoreOverload() StoreCreateOption {
+	return func(store *StoreInfo) {
+		store.overloaded = true
+	}
+}
+
+// SetStoreUnburden allows balancer to select the store.
+func SetStoreUnburden() StoreCreateOption {
```

Comment:
em, how about `ClearStoreOverload` or `ResetStoreOverload`? To make it possible to tell from name to know it is exactly the opposite of `SetStoreOverload`.